### DISCUSSION TITLE
SSH.NET Where does ssh.net store host public keys?

### DISCUSSION THREAD

**Discussion by prettypot** [sshnet/SSH.NET#1518] [General]
Hi,

SSH.NET Where does ssh.net store host public keys? 

Is there an equivalent known_hosts file kept somewhere?

TIA.

**Comment by Rob-Hague** [comment]
The library does not have any kind of known hosts store built in. In order to verify the host identity against an expected value, see [Verify host identity](https://sshnet.github.io/SSH.NET/examples.html#verify-host-identify) example
